test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -63,10 +63,13 @@ public void InitializeShouldNotThrowExceptionIfEventsIsNotNull()
         [TestMethod]
         public void InitializeWithParametersShouldThrowExceptionIfEventsIsNull()
         {
+            var parameters = new Dictionary<string, string>
```

Comment:
> var parameters = new Dictionary<string, string> [](start = 11, length = 48)

Create in test initialize

Reply:
As we are doing it in single shot, there won't be much difference i guess.